### DISCUSSION TITLE
I have implementing chakra UI in my mern stack project  but when I run project onclick it lost connectivity with server and API return nothing but server is connected

### DISCUSSION THREAD

**Discussion by JamimahIram** [chakra-ui/chakra-ui#8798] [Q&A] [answered]
I have created project in mern stack. I am not using any react framework like nextjs ,Gatsby etc. Version installed in my machine, reactjs: 17.0.1 and react-router-dom: 17.0.2 , react-bootstrap: 2.7.0.                                                                                                                                                                           I have implementing chakra UI button successfully but when I run project onclick it lost connection with server and API return nothing even project lost connection.In browser it is showing  no internet as internet is running fine. In network tab API hit but return status code 204 no content.                                                                                                                                                                                                      I have checked my express node server , it is connected. But I don't know why frontend is being disconnected from server. 
At present I have implemented chakra in just one component which is to use save post.                                                                                          I have tried in google, firefox and edge. in google and edge it is not working fine but in firefox mostly it run fine but in firefox also throw error of 204 no content

**Comment by segunadebayo** [comment, marked as answer]
This is not a Chakra bug. A 204 from your API is a backend issue.
